lifecycle-operator/chart/templates/deployment.yaml

```diff
@@ -5,12 +5,9 @@ kind: ServiceAccount
 metadata:
   name: keptn-scheduler
   namespace: {{ .Release.Namespace | quote }}
-  {{- $annotations := include "common.annotations" (dict "context" .) }}
-  {{- with $annotations }}
-  annotations: {{- . -}}
-  {{- end }}
   labels:
 {{- include "common.labels.standard" ( dict "context" . ) | nindent 4 }}
+{{- include "common.annotations" ( dict "context" . ) }}
```

Comment:
that's the way how we add annotations for the other ServiceAccounts and basically in each other resources (where it makes sense). This is just to align it to the rest and use the already implemented functions from the common helm charts

Reply:
I do not think we should change this, the with condition is there to avoid an empty 

annotations : 

In case the common annotations are undefined, in other places where the simple include is there we have some hardcoded annotations

Reply:
fixed